fighters/common/src/function_hooks/mod.rs

```diff
@@ -42,9 +42,9 @@ pub fn install() {
     unsafe {
         // Handles getting rid of the kill zoom
         const NOP: u32 = 0xD503201Fu32;
-        skyline::patching::patch_data(utils::offsets::kill_zoom_regular(), &NOP);
-        skyline::patching::patch_data(utils::offsets::kill_zoom_throw(), &NOP);
+        skyline::patching::Patch::in_text(utils::offsets::kill_zoom_regular()).nop();
+        skyline::patching::Patch::in_text(utils::offsets::kill_zoom_throw()).data(NOP);
```

Comment:
is there a reason one of these uses `.nop()` and the other uses `.data(NOP)`

Reply:
> is there a reason one of these uses `.nop()` and the other uses `.data(NOP)`

The NOP in `.data(NOP)` is actually a reference to an address, defined a few lines above. I’m not sure if it functionally does the same thing as .nop(), but that’s how Blujay had it implemented.

Reply:
it might be wise to rename that variable to REPLACEMENT_DATA or something if we want to leave it unknown. Right now, this is the kind of thing someone might want to just fix in passing... and that could be bad.